lab1_v4.py

```diff
@@ -0,0 +1,34 @@
+import random
+
+N, M = (5, 5)
+
+
+def sum_max_items():
+    max_sum = max(arr[i + j][j] + arr[j][i + j] for i in range(1, N) for j in range(M - i))
+    print('Max sum:', max(max_sum, arr[0][-1]))
+
+
+def multiplication_positive_num():
+    for i, row in enumerate(arr):
+        counter = 1
+        for element in row:
+            if element < 0:
+                print(i + 1, ' row: there is a negative element')
+                break
+            counter *= element
+        else:
+            print('Multiplication of positive numbers of ', i + 1, ' row: ', counter)
```

Comment:
I think comprehension is possible here

Reply:
I can't get a sense of how to do it. Can we review this on the meeting on Tuesday?